src/main/java/org/folio/inventory/dataimport/handlers/actions/UpdateItemEventHandler.java

```diff
@@ -103,6 +102,9 @@ public class UpdateItemEventHandler implements EventHandler {
   private final Storage storage;
   private final MappingMetadataCache mappingMetadataCache;
 
+  private final List<PartialError> resultedErrorItems = new ArrayList<>();
+  private final List<Item> resultedSuccessItems = new ArrayList<>();
```

Comment:
Same concern about being shared between multiple import processes as in UpdateHoldingEventHandler

Reply:
@tatsiana-tarhonskaya  the same as in the comment above.